### PR TITLE
1.0.8

### PR DESCRIPTION
**Pull request by DenBond7**
- [x] ckeck tests

- [x] test on android 6.0, 7.0

### CODE REVIEW

File: FlowCrypt/src/main/java/com/flowcrypt/email/api/email/sync/tasks/DeleteMessagesPermanentlySyncTask.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    val trash = foldersManager.folderTrash
+
+    if (trash == null) {
+      listener.onActionCompleted(account, ownerKey, requestCode)
+      return
+    }
+
+    val roomDatabase = FlowCryptRoomDatabase.getDatabase(context)
+
+    while (true) {
```
Comment (tomholub) [inline FlowCrypt/src/main/java/com/flowcrypt/email/api/email/sync/tasks/DeleteMessagesPermanentlySyncTask.kt:41]: There is some possible chance here for infinite loop - if the remote server didn't actually move or change items as expected. We could consider instead testing either `isEmpty` OR that the amount of items did not change from last cycle.

But if you think there is no chance, then no need.

**Comment by tomholub** [comment]
Is this otherwise ready to release?

**Comment by tomholub** [comment]
Ah sorry, I see it's still marked as a draft

**Comment by DenBond7** [comment]
> Ah sorry, I see it's still marked as a draft

yup, that's still a draft

**Pull request merged**